user/Dockerfile

```diff
@@ -69,17 +69,21 @@ ADD install-base-notebook.bash /usr/local/sbin/install-base-notebook.bash
 RUN /usr/local/sbin/install-base-notebook.bash
 
 ADD install-python-packages.bash /usr/local/sbin/install-python-packages.bash
-ADD connectors /usr/local/sbin/connectors-setup
 RUN /usr/local/sbin/install-python-packages.bash
+# Execute all connector setup scripts
+ADD connectors /usr/local/sbin/connectors-setup
+RUN /bin/bash -c 'ls /usr/local/sbin/connectors-setup/*.bash | xargs -L1 /bin/bash'
+
 
 # Since we put NLTK data in /opt rather than /usr/local,
 # we have to explicitly set this
 ENV NLTK_DATA ${CONDA_DIR}/nltk
 
+USER root
```

Comment:
I moved it up because I was getting a "Permission Denied" on one of the bash commands when trying to rename the downloaded gdrive executable and putting it in /usr/local/bin.